cirq-core/cirq/experiments/single_qubit_readout_calibration.py

```diff
@@ -51,6 +55,121 @@ def _json_dict_(self) -> Dict[str, Any]:
             'timestamp': self.timestamp,
         }
 
+    def plot_heatmap(
+        self,
+        axs: Optional[tuple[plt.Axes, plt.Axes]] = None,
+        annotation_format: str = '0.1%',
+        **plot_kwargs: Any,
+    ) -> tuple[plt.Axes, plt.Axes]:
+        """Plot a heatmap of the readout errors. If qubits are not cirq.GridQubits, throws an error.
+
+        Args:
+            axs: a tuple of the plt.Axes to plot on. If not given, a new figure is created,
+                plotted on, and shown.
+            annotation_format: The format string for the numbers in the heatmap.
+            **plot_kwargs: Arguments to be passed to 'cirq.Heatmap.plot()'.
+        Returns:
+            The two plt.Axes containing the plot.
+
+        Raises:
+            ValueError: axs does not contain two plt.Axes
+            TypeError: qubits are not cirq.GridQubits
+        """
+
+        if axs is None:
+            _, axs = plt.subplots(1, 2, dpi=200, facecolor='white', figsize=(12, 4))
+
+        else:
+            if (
+                not len(axs) != 2 or type(axs[0]) != plt.Axes or type(axs[1]) != plt.Axes
```

Comment:
add `not isinstance(axs, (tuple, list))` for the case where axs are not iterable

```py3
not isinstance(axs, (tuple, list)) or len(axs) != 2 or type(axs[0]) != plt.Axes or type(axs[1]) != plt.Axes
```